docs/_includes/common/tooltip.html

```diff
@@ -181,6 +181,33 @@ <h3 class="infotip__heading">Infotip</h3>
 </p>
     {% endhighlight %}
 
+    <p>Infotip on small screens</p>
```

Comment:
@raveenakothapally Let's also add this demo and code example under the dialog section. Before release I will most likely rename it something a little more generic than `dialog-infotip`, because I see it's potential in places other than infotip, but the current name will suffice for now.